app/templates/tools/includes/list.html

```diff
@@ -57,4 +57,8 @@ <h2 class="heading-medium">Your tools</h2>
     </tbody>
   </table>
 
+  <p>
+    You can <a href="https://grafana.services.alpha.mojanalytics.xyz/d/000000002/platform-users?refresh=10s&amp;orgId=1&amp;var-Username={{ current_user.username }}" target="_blank">view your resource utilisation on Grafana (opens in new tab)</a>.
```

Comment:
As far as I can tell it looks good to me. I'd probably replace the hardcoded environment at least with the environment name from the config.

(You may want to add a separate config for the whole grafana URL but that's probably overkill for now as you can just derive it from what you already have)

Reply:
You think it's worth urlencode the username? Probably it should be fine.

Reply:
We don't have an equivalent `platform-users` dashboard in Grafana on dev, so the link wouldn't work.

Reply:
I see, I think would be a bit cleaner maybe if the URL is in the config, but up to you really,